EntAppProject/src/main/resources/templates/JBCreateAccount.html

```diff
@@ -23,18 +23,18 @@ <h2>Create an Account</h2>
 <form action="/JBHomescreen" method="post">
 
 <div class="container">
-        <label for="useremail"><b>Email:</b></label>

```

Comment:
For attributes in labels need to match the input's ID that it matches.

For example:

This is valid:
```
        <label for="username"><b>Username:</b></label>
        <input type="text" placeholder="Username:" name="username" required>
```

This is not: 

```
        <label for="userpassword"><b>Password:</b></label>	       
        <input type="password" placeholder="Password" name="userpassword" required>
```